app/models/lookup/import/adviser.rb

```diff
@@ -0,0 +1,26 @@
+module Lookup
+  module Import
+    class Adviser < ActiveRecord::Base
+      validates_length_of   :reference_number, is: 8
+      validates_presence_of :name
+
+      def self.table_name
+        "lookup_import_#{super}"
+      end
+
+      def self.fca_import_copy_statement
+        "COPY #{table_name} (reference_number, name, created_at, updated_at) FROM stdin;"
+      end
+
+      def self.truncate_sql
+        "TRUNCATE #{table_name};"
```

Comment:
We can access a Rails provided way of truncating tables:

https://github.com/rails/rails/blob/v4.2.5.2/activerecord/lib/active_record/connection_adapters/postgresql_adapter.rb#L261

```ruby
ActiveRecord::Base.connection.truncate(table_name)
```